src/app/(nextjs_migration)/(authenticated)/user/welcome/page.tsx

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Image from "next/image";
+import Script from "next/script";
+
+import { getL10n } from "../../../../functions/server/l10n";
+import { getLocale } from "../../../../../utils/fluent.js";
+
+export async function generateMetadata() {
+  const l10n = getL10n();
+  return {
+    title: "Welcome", // FIXME l10n.getString("breach-meta-title"),
+    twitter: {
+      card: "summary_large_image",
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      images: ["/images/og-image.webp"],
+    },
+    openGraph: {
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      siteName: l10n.getString("brand-fx-monitor"),
+      type: "website",
+      url: process.env.SERVER_URL,
+      images: ["/images/og-image.webp"],
+    },
+  };
+}
+
+export default async function UserBreaches() {
+  const l10n = getL10n();
+
+  return (
+    <>
+      <main>
+        <h1>Welcome to Monitor. Let’s find your exposed information.</h1>
+        <p>
+          To find and fix your exposed data, we’ll need to ask for your personal
+          info. Rest assured we will safeguard your information like it’s our
+          own.
+        </p>
+
+        <p>
+          We believe in your right to privacy, so basic Monitor is always free.
+          Upgrade to Premium for continuous protection and automatic removal of
+          your personal info.
+        </p>
+
+        <p>See how we protect your data</p>
```

Comment:
Should this have a period, or be a link? It feels a bit funny here.

Reply:
Yeah I don't have anything to link it to, will remove for now thanks.